ovs/dal/hybrids/albabackend.py

```diff
@@ -407,6 +408,7 @@ def _load_backend_info(_connection_info, _alba_backend_guid):
                     return_value[_alba_backend_guid]['live_status'] = info['live_status']
             except NotFoundException:
                 return_value[_alba_backend_guid]['error'] = 'backend_deleted'
+                return_value[_alba_backend_guid]['live_status'] = 'failure'
```

Comment:
Although I would have liked to see some commitment and usage of the python enum (what enum in python :O), I approve.